samples/IISSample/project.json

```diff
@@ -23,5 +23,11 @@
     "include": [
       "web.config"
     ]
+  },
+  "tools": {
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-preview2-final"
```

Comment:
Should be actually 1.1.0-preview4-final for 1.1 runtime, but I would imagine 1.1.0-* should work?